aboutcitus/what_is_citus.rst

```diff
@@ -1,6 +1,41 @@
 What is Citus?
 ==============
 
-Citus horizontally scales PostgreSQL across commodity servers using sharding and replication. Its query engine parallelizes incoming SQL queries across these servers to enable real-time responses on large datasets.
+Citus horizontally scales PostgreSQL across multiple machines using sharding and replication. It works well when you have a large data set and when you want to get answers from that data in human real-time – typically in less than a second.
```

Comment:
Any specific reason why we removed "parallelize". I think that is one of our bigger value propositions and seems like an important thing to mention.